tree-items.txt

```diff
@@ -506,14 +506,17 @@ xfs/ext4). In BTRFS a file extent can be in one of 3 states:
 
 * BTRFS_FILE_EXTENT_INLINE - an extent item who can hold all of the actual file
 data in its body is considered an inline extent. In such cases data is read
-directly from the body of the extent.
+directly from the body of the extent. The current default value is 2048 bytes,
+but it can be changed with max_inline mount option. The max_inline value is then
+calculated with min(max_inline_value, sectorsize).
 
 * BTRFS_FILE_EXTENT_PREALLOC - a pre-allocated extent. These are ranges which
-have been reserved by using fallocate.
+have been reserved by using fallocate. Each extent can map up to 256Mb for
+pre-allocated extents.
 
 * BTRFS_FILE_EXTENT_REG - regular file extent. Most of the extents of a file
 would be regular extents. Those extents hold information necessary to locate
-the actual data on disk.
+the actual data on disk. Each extent can map up to 128Mb for regular extents.
```

Comment:
That's also correct but still, I think it's going too deep into implementation detail which can change at any time.

Reply:
Following your explanation on IRC I'm ok with this change.

Reply:
Same reason as above. As this repo is a "dev-docs" I would expect that we can visit this repo quite frequently (I do at least), and I see no reason to why not have the extent mapping size not written at any place. The same thing can be said about btrfs.wiki, which also has info that "can change at anytime" IMHO